root/relationship/linktype/RelationshipTypeIndex.js

```diff
@@ -0,0 +1,232 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {sortBy, upperFirst} from 'lodash';
+
+import {withCatalystContext} from '../../context';
+import Layout from '../../layout';
+import Cardinality from '../../static/scripts/common/components/Cardinality';
+import OrderableDirection
+  from '../../static/scripts/common/components/OrderableDirection';
+import Relationship
+  from '../../static/scripts/common/components/Relationship';
+import linkedEntities from '../../static/scripts/common/linkedEntities';
+import expand2react from '../../static/scripts/common/i18n/expand2react';
+import formatEntityTypeName
+  from '../../static/scripts/common/utility/formatEntityTypeName';
+
+type Props = {
+  +$c: CatalystContextT,
+  +relType: LinkTypeT,
+};
+
+const RelationshipTypeIndex = ({
+  $c,
+  relType,
+}: Props) => {
+  const childrenTypes = relType.children || [];
+  const typeName = upperFirst(l_relationships(relType.name));
+  const title = l('Relationship Type') + ' / ' + typeName;
+  const type0 = relType.type0;
+  const type1 = relType.type1;
+  const formattedType0 = formatEntityTypeName(type0);
+  const formattedType1 = formatEntityTypeName(type1);
+  const possibleAttributes = [];
+  Object.keys(relType.attributes)
+    .map(attributeTypeId => (
+      possibleAttributes.push(
+        linkedEntities.link_attribute_type[parseInt(attributeTypeId, 10)],
+      )
+    ));
```

Comment:
The callback passed to `map` will convert each value for you, so you can simplify this a bit.
```suggestion
  const possibleAttributes = Object.keys(relType.attributes).map(attributeTypeId => (
    linkedEntities.link_attribute_type[parseInt(attributeTypeId, 10)]
  ));
```